solution,status,score,%test passed,time,memory,subtask_0,subtask_1,subtask_2,subtask_3
dizalo_gpt-4.1_0.cpp,TIME_LIMIT_EXCEEDED,0,0.00%,3.310,3.66796875,0,0,0,0
dizalo_gpt-4.1_1.cpp,RUNTIME_ERROR,0,0.00%,0.080,5.9375,0,0,0,0
dizalo_gpt-4.1_2.cpp,RUNTIME_ERROR,0,0.00%,0.070,4.94921875,0,0,0,0
dizalo_gpt-4.1_3.cpp,TIME_LIMIT_EXCEEDED,0,0.00%,3.310,6.453125,0,0,0,0
dizalo_gpt-4.1_4.cpp,RUNTIME_ERROR,0,0.00%,0.070,3.36328125,0,0,0,0
dizalo_gpt-4.1_5.cpp,TIME_LIMIT_EXCEEDED,0,0.00%,3.310,4.390625,0,0,0,0
dizalo_gpt-4.1_6.cpp,TIME_LIMIT_EXCEEDED,0,0.00%,3.340,4.328125,0,0,0,0
dizalo_gpt-4.1_7.cpp,TIME_LIMIT_EXCEEDED,0,0.00%,3.310,3.62109375,0,0,0,0
